### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Edit-Web Application_Keyboard: Focus is not visible on any of the interactive controls of the page, while navigating through the keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#163]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated**: MAS2.4.7

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "More" commands button of the existing website inside the grid and Press Enter.
6. Tab till "Edit" in the expanded menu and Press Enter.
7. Tab till "Web Applications" in the left menu section and Press Enter.
8. Tab till "Create Web Application" button and Press Enter.
9. Navigate to Application Pool page tab.
10. Navigate to all the interactive controls of the page and verify that Keyboard focus is clearly visible on every control.

**Actual Result:**
Keyboard Focus is not visible on the following controls of the page:-
"Create Web Application" button, "Use Custom Application Pool" & "Auto Start" toggle button, "Default App Pool" action button, "Pipeline" & ".Net Framework" list items.

**Expected Result:**
When any interactive control receives keyboard focus, a rectangular border should appear around these controls so that user can differentiate clearly between the focused and non-focused controls on the page.

**Note:**- Similar Issue is observed in "Virtual directories" Page, where keyboard focus is not visible on "Create Virtual Directory" button, "Custom Identity" toggle button, "Logon Method" list items.

**User Impact**
This Success Criterion helps anyone who relies on the keyboard to operate the page, by letting them visually determine the component on which keyboard operations will interact at any point in time.
People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
![bug 163_focus visibility issue-1](https://user-images.githubusercontent.com/38521133/44787706-7975e580-abb5-11e8-8856-b3491a0065b0.png)
![bug 163_focus visibility issue-2](https://user-images.githubusercontent.com/38521133/44787707-7975e580-abb5-11e8-8283-1e6a053088bd.png)

**Comment by jhkimnew** [comment]
Fixed. 
Logon Method is a different issue. We can't set input focus on the buttons of Logon Methods. That will be fixed later.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
